.github/workflows/config.yml

```diff
@@ -0,0 +1,35 @@
+# This workflow will do a clean installation of node dependencies, cache/restore them, build the source code and run tests across different versions of node
+# For more information see: https://docs.github.com/en/actions/automating-builds-and-tests/building-and-testing-nodejs
+
+name: RideOn-frontend
+
+on:
+  push:
+    branches: [ "main", "*", "*/*", "**" ]
+  pull_request:
+    branches: [ "main" ]
+
+jobs:
+  build-and-test:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [16.x]
```

Comment:
Lets restrict the node engine version in the package.json as well. So that there is no conflict in individual developer machines and the pipeline